lib/MusicBrainz/Server/Controller/Release.pm

```diff
@@ -94,20 +95,14 @@ after 'load' => sub {
     my $cdtoc_count = $c->model('MediumCDTOC')->find_count_by_release($release->id);
     $c->stash->{release_cdtoc_count} = $cdtoc_count;
 
-    # We need to load more artist credits in 'show'
-    if ($c->action->name ne 'show') {
-        $c->model('ArtistCredit')->load($release);
-    }
+    $c->model('ArtistCredit')->load($release, $release->release_group);
+    $c->model('ReleasePackaging')->load($release);
+    $c->model('ReleaseStatus')->load($release);
+    $c->model('Language')->load($release);
+    $c->model('Script')->load($release);
+    $c->model('Release')->load_related_info($release);
```

Comment:
Mediums are loaded by `load_related_info`. They were already loaded for every action except `edit` (see below).